.github/workflows/ci.yaml

```diff
@@ -57,6 +57,12 @@ jobs:
       - name: Start services
         run: docker-compose -f docker-compose.test.yaml up -d
 
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        with:
+          version: v0.10.0
+          node_image: kindest/node:v1.19.7@sha256:a70639454e97a4b733f9d9b67e12c01f6b0297449d5b9cbbef87473458e26dca
```

Comment:
It's probably fine for now, but we should probably move kube test running to a separate job and use a test matrix to run tests on multiple kube versions. Can you open an issue for that?

Also, see the helm-chart repo for an example.

Reply:
Thanks! Yeah, that is a good idea. I thought that we could start with something simple.